build/Packaging/NugetAgent/NewRelic.Agent.nuspec

```diff
@@ -5,10 +5,10 @@
     <version>TO_BE_REPLACED</version>
     <title>New Relic .NET Agent</title>
     <authors>New Relic</authors>
-    <owners>New Relic</owners>
     <license type="expression">Apache-2.0</license>
     <projectUrl>https://github.com/newrelic/newrelic-dotnet-agent</projectUrl>
     <iconUrl>https://newrelic.com/static-assets/images/icons/avatar-newrelic.png</iconUrl>
```

Comment:
Get rid of iconUrl... it's actually a security vulnerability since it generates web requests whenever a smart client displays the package info

Reply:
Microsoft's docs indicate that this is not an issue and they have a tip that mentions retaining the link for backwards compatibility. I would prefer to error on the side of caution. I don't think this is a security issue since its an asset we control and   Microsoft would not have the tip about retaining it if it was.  See the tip at the end of the section: https://learn.microsoft.com/en-us/nuget/reference/nuspec#icon